cluster/ubuntu/util.sh

```diff
@@ -364,6 +369,11 @@ function provision-master() {
     ubuntu/binaries/master/ \
     "${MASTER}:~/kube"
 
+  # copy the flannel binaries and scripts, if flannel is being used
```

Comment:
Where is the previous copy? I don't see that being changed/removed in this PR....